spec/pdfs/distribution_pdf_spec.rb

```diff
@@ -26,8 +26,8 @@
       expect(results).to eq([
         ["Items Received", "Requested", "Received", "Value/item", "In-Kind Value Received", "Packages"],
         ["Item 1", "", 50, "$1.00", "$50.00", "1"],
-        ["Item 2", 30, 100, "$2.00", "$200.00", nil],
-        ["Item 3", 50, "", "$3.00", nil, nil],
+        ["Item 2", "30", 100, "$2.00", "$200.00", nil],
```

Comment:
These are surprisingly stored as strings; in the old one they ended up number-ified somewhere, but they are turned into strings in the PDF anyway so strings are fine.